src/App.js

```diff
@@ -85,47 +86,15 @@ class App extends React.Component {
 
   calculate = () => {
     const { result, operate, originalLastNum } = this.state;
-    let finalResult = 0;
-    var tempResult = result;
 
-    [tempResult, finalResult] = cleanupEquation(
+    var [tempResult, finalResult] = cleanupEquation(
       result,
       operate,
       originalLastNum
     );
 
-    var tempResultString = tempResult;
-    //We need to detect the non nested parenthesis
-    //Open ( and no other ( before the next close
-    // '(' 0123456789-+*/.(not (, not ) ) ')'
-    // ( + any number of the 15 chars + ) : (7+2) (-3.25478/+6.587)
-    const reg = /\(([0123456789/*-+.]*)\)/g;
-    var parenthesisToCalculate = tempResultString.match(reg) || [];
-    while (parenthesisToCalculate.length > 0) {
-      for (var i = 0; i < parenthesisToCalculate.length; i++) {
-        //we extract the first expression ex:  (2+5)
-        let expression = parenthesisToCalculate[i];
-        //We calculate the value ex: 7
+    handleNestedParens(tempResult);
```

Comment:
Why is this being called here? Doesn't appear that we are using the return value.

Reply:
True. This is from long ago so not sure what commit this was a part of as github just batches all these comments on this old App.js file that is now outdated.

Reply:
This is part of the commit titled "extract handleDoubleEqual and handleNestedParens"